cli/src/secret.rs

```diff
@@ -1,34 +1,46 @@
 use rust_dotenv::dotenv::DotEnv;
 
+const FILENAME: &str = ".env.secrets";
+const PREFIX: &str = "KINETICS_SECRET_";
+
 pub struct Secret {
     pub name: String,
     value: String,
 }
 
 impl Secret {
-    /// Read secrets from the .env file
-    pub fn from_dotenv() -> Vec<Self> {
-        // Return early if the .env.secrets file doesn't exist,
-        // to avoid rust_dotenv throwing error message
-        if !std::path::Path::new(".env.secrets").exists() {
-            log::warn!("No .env.secrets file found");
-            return vec![];
-        }
-
-        let mut result = vec![];
-        let dotenv = DotEnv::new("secrets");
-
-        for (name, value) in dotenv.all_vars() {
-            result.push(Secret {
-                name: name.clone(),
-                value: value.clone(),
-            });
+    /// Read secrets from the .env file or env vars if file not found.
+    pub fn load() -> Vec<Self> {
+        if !std::path::Path::new(FILENAME).exists() {
+            log::warn!(
+                "No .env.secrets file found. Search for {PREFIX} prefixed environment variables."
+            );
+            return std::env::vars()
+                .filter(|(name, _)| name.starts_with(PREFIX) && name != PREFIX)
```

Comment:
@quesabe I'm now wondering if we really need the prefix. It actually will make CI/CD use more difficult, because we need to explain the prefix to the user. I can clearly see them adding secrets in a way they usually add them and them being frustrated by kinetics not being able to find the secrets 😬

Let's get rid of the prefix, and instead filter out env vars which may expose some sensitive OS data to the process. What do you think?